app/controllers/caseload_controller.rb

```diff
@@ -3,19 +3,21 @@
 class CaseloadController < PrisonsApplicationController
   before_action :ensure_pom
 
-  breadcrumb -> { 'Your caseload' }, -> { prison_caseload_index_path(active_prison) }
+  breadcrumb -> { 'Your caseload' },
+             -> { prison_caseload_index_path(active_prison) }
   breadcrumb -> { 'New cases' },
              -> { new_prison_caseload_path(active_prison) }, only: [:new]
+  breadcrumb -> { 'Cases close to handover' },
```

Comment:
These breadcrumbs look a little strange  - we seem to have a link to ourselves (even for new) which looks wrong. I thought the idea was to show links backwards, not including the page that we are actually on.

Reply:
Think we need to investigate whether the gem let's us define our own views so that last is inactive.